src/components/vm/hostdevs/hostDevAdd.jsx

```diff
@@ -88,10 +88,10 @@ const DevRow = ({ idPrefix, type, selectableDevices, setSelectableDevices }) =>
             cells.push(getOptionalValue(device, `${id}-device`, _("Device")));
             cells.push(getOptionalValue(bus, `${id}-bus`, _("Bus")));
         } else if (nodeDev.capability.type === "pci") {
-            let domain = nodeDev.capability.domain;
-            let bus = nodeDev.capability.bus;
-            let slot = nodeDev.capability.slot;
-            let func = nodeDev.capability.function;
+            let domain = Number(nodeDev.capability.domain);
+            let bus = Number(nodeDev.capability.bus);
+            let slot = Number(nodeDev.capability.slot);
+            let func = Number(nodeDev.capability.function);
```

Comment:
This is an unrelated change -- can you please move it to a separate commit?

Reply:
Ahhh, how this got here, this should not even be in his PR

Reply:
fixed